lib/screens/home_screen.dart

```diff
@@ -76,4 +93,47 @@ class _HomeScreenState extends State<HomeScreen> {
       });
     }
   }
+
+  _showbottomsheet(BuildContext context) {
+    showModalBottomSheet(
+      context: context,
+      builder: (_) => model == null
```

Comment:
handle all possible cases when there will be no files i.e _model.files==null as well as _model.files.length==0